packages/expo-camera/src/next/Camera.types.ts

```diff
@@ -94,7 +94,7 @@ export type CameraCapturedPicture = {
 // @needsAudit
 export type CameraPictureOptions = {
   /**
-   * Specify the quality of compression, from 0 to 1. 0 means compress for small size, 1 means compress for maximum quality.
+   * Specify the quality of compression, from `0` to `1`. `0` means compress for small size, `1` means compress for maximum quality.
```

Comment:
```suggestion
   * Specify the compression quality from `0` to `1`. `0` means compress for small size, and `1` means compress for maximum quality.
```

Unrelated to the change in the PR, missing conjunction.